index.html

```diff
@@ -197,6 +197,46 @@ <h3 class="frameworks">Skills</h3>
         </div>
     </section>
 
+    <section class="contact-form">
+        <p>
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </p>
+        <form action="https://formspree.io/f/mvonwygw" method="POST">
+            <input type="text" name="full_name" maxlength="30" placeholder="Full name" required/>
+            <input type="email" name="email" placeholder="Email" required/>
+            <textarea name="textarea" maxlength="500" placeholder="Write me something..." required></textarea>
+            <input type="button" title="Get in touch" placeholder="Get in touch" />
+        </form>
+        <ul class="social_link">
+            <li></li>
+        </ul>
+    </section>
```

Comment:
- Your contact form is not styled at all. Kindly style it to match the original design.

![image](https://user-images.githubusercontent.com/98453979/230149592-46e37534-de6f-4d1a-94d2-f0ccebb214c0.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/230149880-d1647b39-149c-4f9d-a694-5cf7d5e3b554.png)